### DISCUSSION TITLE
Is is possible to make an extension that creates a new panel?

### DISCUSSION THREAD

**Discussion by AtomAlpaca** [zed-industries/zed#17602] [Feature Requests]
I am trying to develop a agda extension for zed and I met some difficulties.

In the agda-mode extension of emacs, we can hit a hotkey and it appears an interactive panel with proof goal on it, I wonder:

1. [Is it possible to make an extension with hotkeys?](https://github.com/zed-industries/zed/discussions/17557)
2. Is it possible to make an extension that creates a new panel?

I found nothing related in the docs and there is no other extension has such features.

**Comment by SomeoneToIgnore** [comment]
It's not possible to do much in extensions now, alas, beyond dealing with highlights, LSP and providing slash commands with tasks.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
